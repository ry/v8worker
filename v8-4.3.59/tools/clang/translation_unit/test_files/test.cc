// Copyright (c) 2014 The Chromium Authors. All rights reserved.
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

#include "test.h"

#include <bits/wchar.h>

#include "binomial.h"

// Notice that "binomial.h" is included both here and in the "test.h" file.
// The tool should however print the path to this header file only once.

int main() {
  // Just some nonesense calculations.
  int result = calculateNumberOfWaysToDistributeNItemsAmongKPersons(10, 5);
  return result + binomial(42, 1);
}
