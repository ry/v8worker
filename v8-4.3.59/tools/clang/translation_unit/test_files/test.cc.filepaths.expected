bits/wchar.h
./binomial.h
./test.cc
./test.h
